### ISSUE TITLE
Retroactive Tick Problems

### ISSUE THREAD

**Issue by Ninja9000** [hackagotchi/hackagotchi#78]
Yields coming after crafts that destroy plants still go through with how retroactive ticks work.
![One](https://user-images.githubusercontent.com/25556133/85963603-75b77380-b984-11ea-9f2b-bec5493cf19a.png)
![Two](https://user-images.githubusercontent.com/25556133/85963604-76500a00-b984-11ea-84b7-a17e19d15868.png)

**Comment by Ninja9000** [comment]
You could also potentially use a Warp to have multiplied ticks if using and going inactive, due to the Warp duration being over the 5 minutes a Hackstead takes to stop updating.

**Comment by lukec11** [comment]
Yep we tested this, unfortunately the warp powder bug is a problem.
